### ISSUE TITLE
Turn of intermediate pre-fetching

### ISSUE THREAD

**Issue by swghosh** [tensorflow/datasets#2316]
https://github.com/tensorflow/datasets/blob/bdf30a10adbedac843374052d81e9bae607fd9b3/tensorflow_datasets/core/dataset_builder.py#L572

Is it possible to turn off prefetching when use `tfds.load(ds_name)`?

**Comment by Conchylicultor** [comment]
I don't think it is possible currently, but please don't hesitate to send a PR to add the option in `tfds.ReadConfig`

**Comment by swghosh** [comment]
@Conchylicultor I'll work on it.

**Comment by Conchylicultor** [comment]
Thank you! For the name maybe `skip_prefetch: bool = False` ?

**Comment by swghosh** [comment]
That should be good.
